woocommerce/woocommerce-android
# [Dynamic Dashboard] Extract Blaze Card

Part of: #11250 


### Description
This PR just extracts all logic of Blaze card to the Composable function to be ready for the dynamic dashboard.

### Testing instructions
1. Using a Jetpack or WPCom website, open the app.
2. Confirm the Blaze card is shown (assuming it was not dismissed before).
3. Launch campaign creation, and confirm navigation works.
4. Dismiss the card.
5. Confirm the view gets hidden.
